task-1-STREAMS/sources/Student.java

```diff
@@ -0,0 +1,15 @@
+class Student{
+
+    Map<String, Integer> rating;
```

Comment:
think the fields should be private

Reply:
this task was from Ilya, so I've just copied it

Reply:
Maybe fix it anyway because of best practices and java convention reasons